db/migrate/20230815163236_create_comments.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Migration to create the Comment table.
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: { to_table: :users }
```

Comment:
* As stated in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md#:~:text=Be%20sure%20to%20reference%20the%20ERD%20diagram%20from%20the%20sneak%20peek%20if%20you%20have%20any%20questions%20about%20table%20or%20column%20names%20or%20data%20relationships.): Be sure to reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships.   as currently, the comments table have a different column name as shown in the ERD below 👇 
![image](https://github.com/momo-87/blog-app/assets/52497246/0e08043b-90d9-4368-8d7a-1990cea5e834)